src/wp-includes/class-wp-block-parser.php

```diff
@@ -5,162 +5,6 @@
-	 *   'innerBlocks'  => array( block, block ),
-	 *   'innerContent' => array( 'Before', null, 'Inner', null, 'After' ),
-	 * )
-	 *
-	 * @since 4.2.0
-	 * @var array
-	 */
-	public $innerContent;
-
-	/**
-	 * Constructor.
-	 *
-	 * Will populate object properties from the provided arguments.
-	 *
-	 * @since 5.0.0
-	 *
-	 * @param string $name         Name of block.
-	 * @param array  $attrs        Optional set of attributes from block comment delimiters.
-	 * @param array  $innerBlocks  List of inner blocks (of this same class).
-	 * @param string $innerHTML    Resultant HTML from inside block comment delimiters after removing inner blocks.
-	 * @param array  $innerContent List of string fragments and null markers where inner blocks were found.
-	 */
-	public function __construct( $name, $attrs, $innerBlocks, $innerHTML, $innerContent ) {
-		$this->blockName    = $name;
-		$this->attrs        = $attrs;
-		$this->innerBlocks  = $innerBlocks;
-		$this->innerHTML    = $innerHTML;
-		$this->innerContent = $innerContent;
-	}
-}
-
-/**
- * Class WP_Block_Parser_Frame
- *
- * Holds partial blocks in memory while parsing
- *
- * @internal
- * @since 5.0.0
- */
-class WP_Block_Parser_Frame {
-	/**
-	 * Full or partial block
-	 *
-	 * @since 5.0.0
-	 * @var WP_Block_Parser_Block
-	 */
-	public $block;
-
-	/**
-	 * Byte offset into document for start of parse token
-	 *
-	 * @since 5.0.0
-	 * @var int
-	 */
-	public $token_start;
-
-	/**
-	 * Byte length of entire parse token string
-	 *
-	 * @since 5.0.0
-	 * @var int
-	 */
-	public $token_length;
-
-	/**
-	 * Byte offset into document for after parse token ends
-	 * (used during reconstruction of stack into parse production)
-	 *
-	 * @since 5.0.0
-	 * @var int
-	 */
-	public $prev_offset;
-
-	/**
-	 * Byte offset into document where leading HTML before token starts
-	 *
-	 * @since 5.0.0
-	 * @var int
-	 */
-	public $leading_html_start;
-
-	/**
-	 * Constructor
-	 *
-	 * Will populate object properties from the provided arguments.
-	 *
-	 * @since 5.0.0
-	 *
-	 * @param WP_Block_Parser_Block $block              Full or partial block.
-	 * @param int                   $token_start        Byte offset into document for start of parse token.
-	 * @param int                   $token_length       Byte length of entire parse token string.
-	 * @param int                   $prev_offset        Byte offset into document for after parse token ends.
-	 * @param int                   $leading_html_start Byte offset into document where leading HTML before token starts.
-	 */
-	public function __construct( $block, $token_start, $token_length, $prev_offset = null, $leading_html_start = null ) {
-		$this->block              = $block;
-		$this->token_start        = $token_start;
-		$this->token_length       = $token_length;
-		$this->prev_offset        = isset( $prev_offset ) ? $prev_offset : $token_start + $token_length;
-		$this->leading_html_start = $leading_html_start;
-	}
-}
-
```

Comment:
When moving classes to their own file in the past, the convention has been to include a `require_once` in the old location to preserve backward compatibility for plugins loading the file directly, see the modifications to `class-wp-customize-control.php` in b4c270030f5dd7a914975dd0aaa05ff2086129bd as an example.

Reply:
Fixed.